code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -17,10 +22,36 @@
 	payday_modifier = 1
 
 /datum/species/human/prepare_human_for_preview(mob/living/carbon/human/human)
-	human.hairstyle = "Business Hair"
-	human.hair_color = "#bb9966" // brown
+	human.hairstyle = "Unkept"
+	human.hair_color = "#f08e33" // brown
+
+	var/obj/item/organ/internal/ears/fox/fox_ears = human.getorgan(/obj/item/organ/internal/ears/fox)
+	if (fox_ears)
+		fox_ears.color = human.hair_color
+		human.update_body()
+
 	human.update_body_parts()
 
+/datum/species/human/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		// Force everyone to have at least fox features. I am evil and fucked up. We don't want to
+		// override cat ears, so I'll do it like this. If you don't like it sue me.
+		if(H.dna.features["tail_cat"] == "None")
+			H.dna.features["tail_cat"] = "Fox"
+		if(H.dna.features["ears"] == "None")
+			H.dna.features["ears"] = "Fox"
+
+		if(H.dna.features["ears"] == "Fox")
+			var/obj/item/organ/internal/ears/fox/ears = new
+			ears.Insert(H, drop_if_replaced = FALSE)
```

Comment:
This is awful, we have an organ generation framework for this shit

Reply:
I had just about 10 minutes to remove merge conflicts before the next round started, actual good code comes later